js/SearchForm.js

```diff
@@ -0,0 +1,109 @@
+class SearchForm {
+  constructor(element, instance) {
+    this.element = element;
+    this.instance = instance;
+    this.searchInputContainer;
+    this.autoSearchDisplay;
+    this.button;
+    this.timer = 0;
+  }
+
+  createForm = () => {
+    this.searchInputContainer = document.createElement("div");
+    CONFIG.userInquiry = document.createElement("input");
+    this.autoSearchDisplay = document.createElement("div");
+    this.button = document.createElement("button");
+    this.element.setAttribute("id", "form");
+    this.searchInputContainer.classList.add("searchInputContainer");
+    CONFIG.userInquiry.classList.add("userSearchField");
+    this.autoSearchDisplay.classList.add("autoSearchQuery");
+    this.button.classList.add("searchButton");
+    CONFIG.userInquiry.setAttribute("placeholder", "Search for Data");
+    this.button.innerText = "Submit";
+    this.element.appendChild(this.searchInputContainer);
+    this.element.appendChild(this.button);
+    this.searchInputContainer.appendChild(CONFIG.userInquiry);
+    this.searchInputContainer.appendChild(this.autoSearchDisplay);
+    this.enableEventListeners(CONFIG.userInquiry);
+  };
+
+  autoSearch = (e) => {
```

Comment:
Although this is not what you ere supposed to do, it's a good thing you practiced buliding this kind of search (: